lib/enums/query/query_presets.py

```diff
@@ -1,9 +1,14 @@
-from enum import Enum, auto
+from enum import auto
+from enums.query.enum_with_aliases import EnumWithAliases
 from exceptions.parse_query_error import ParseQueryError
 
+# pylint: disable=too-few-public-methods
 
-class QueryPresets(Enum):
-    pass
+
+class QueryPresets(EnumWithAliases):
+    @staticmethod
+    def _get_aliases():
+        pass
```

Comment:
We should drop this, and also make this an abstract class

Currently if I made a new class based on `QueryPresets` fails to implement this method it won't error until runtime, whereas dropping this means we couldn't instantiate the new class